content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -168,212 +165,161 @@ To define the `Config` trait for the proof-of-existence pallet:
 Now that you've configured the pallet to emit events, you are ready to define those events.
 As described in [Design the application](#design-the-application), the proof-of-existence pallet emits an event under the following conditions:
 
-- When a new proof is added to the blockchain.
-- When a proof is revoked.
+- When a new claim is added to the blockchain.
+- When a claim is revoked.
 
 Each event also displays an `AccountId` to identify who triggered the
-event and the proof-of-existence data (as `Vec<u8>`) that is being stored or removed.
-By convention, each event includes an array with descriptive names for its parameters.
+event and the proof-of-existence claim (as `Hash`) that is being stored or removed.
 
 To implement the pallet events:
 
 1. Open the `pallets/template/src/lib.rs` file in a text editor.
 
 1. Replace the `#[pallet::event]` line with the following code block:
 
-   ```rust
-   // Pallets use events to inform users when important changes are made.
-   // Event documentation should end with an array that provides descriptive names for parameters.
-   #[pallet::event]
-   #[pallet::generate_deposit(pub(super) fn deposit_event)]
-   pub enum Event<T: Config> {
-   	/// Event emitted when a proof has been claimed. [who, claim]
-   	ClaimCreated(T::AccountId, Vec<u8>),
-   	/// Event emitted when a claim is revoked by the owner. [who, claim]
-   	ClaimRevoked(T::AccountId, Vec<u8>),
-   }
-   ```
+  ```rust
+  // Pallets use events to inform users when important changes are made.
+  // Event documentation should end with an array that provides descriptive names for parameters.
+  #[pallet::event]
+  #[pallet::generate_deposit(pub(super) fn deposit_event)]
+  pub enum Event<T: Config> {
+    /// Event emitted when a proof has been claimed.
+    ClaimCreated { who: T::AccountId, claim: T::Hash },
+    /// Event emitted when a claim is revoked by the owner.
+    ClaimRevoked { who: T::AccountId, claim: T::Hash },
+  }
+  ```
 
 1. Save your changes.
 
-## Include `sp-std` library
-
-You might notice that the proof-of-existence pallet uses the `Vec<u8>` type.
-This type is included in the `std` Rust library.
-However, you _cannot_ use the `std` library for pallet development.
-Instead, the proof-of-existence pallet uses the [sp-std crate](https://paritytech.github.io/substrate/master/sp_std/index.html) to declare the `Vec<u8>` type under the `mod pallet` section:
-
-```rust
-use sp_std::vec::Vec;
```

Comment:
Because we don't use vec, we don't need this section anymore.

thank goodness.